jscripts/bbcodes_sceditor.js

```diff
@@ -529,6 +534,7 @@ $(document).ready(function($) {
 						'<option value="veoh">' + editor._('Veoh') + '</option>' +
 						'<option value="vimeo">' + editor._('Vimeo') + '</option>' +
 						'<option value="youtube">' + editor._('Youtube') + '</option>' +
+						'<option value="twitch">' + editor._('Twitch') + '</option>' +
```

Comment:
Though it doesn't seem to have that much of an impact this variable should also be added to `global.lang.php` as `editor_twitch` like the rest  to make it translatable.